core/src/main/scala/scala/pickling/PicklingErrors.scala

```diff
@@ -0,0 +1,165 @@
+package scala.pickling
+
+private[pickling] object Feedback {
+
+  def failedGenerationMsg(x: String, y: String, z: String) =
```

Comment:
I'll address this in a follow-up PR. Sorry! I was very haskell-ish (using a lazy-strategy).